docs/general/build/smart-contracts/gas-optimization/payable-constructor.md

```diff
@@ -0,0 +1,35 @@
+---
+displayed_sidebar: generalSidebar
+---
+
+# Efficient Constructor Usage
```

Comment:
I would suggest rename the article such as `payable constructor` to precisely describle the issue